libs/cache/tests/unit/local_mru_cache.cpp

```diff
@@ -4,8 +4,8 @@
 //  Distributed under the Boost Software License, Version 1.0. (See accompanying
 //  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-#include <hpx/cache/entries/lru_entry.hpp>
-#include <hpx/cache/local_cache.hpp>
+#include <hpx/util/cache/entries/lru_entry.hpp>
+#include <hpx/util/cache/local_cache.hpp>
```

Comment:
This is most probably not an intended change.

Reply:
indeed !